spec/commands/get_review_recommendations_spec.rb

```diff
@@ -102,7 +102,7 @@ def perform_with_predefined_args
 
   context 'requests older than 1 month have been made' do
     let(:other_users_ids) { [other_user1.id, other_user2.id, other_user3.id, other_user4.id] }
-    let!(:pull_request_1) { create(:pull_request_relation, github_user_id: github_user_id, target_user_id: other_user4.id, gh_updated_at: Time.now - 2.months) }
+    let!(:pull_request_1) { create(:pull_request_relation, github_user_id: other_user4.id, target_user_id: github_user_id, gh_updated_at: Time.now - 2.months) }
```

Comment:
Line is too long. [160/100]<br>Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.